components/PageControl/examples/PageControlTypicalUseExample.swift

```diff
@@ -29,10 +29,10 @@ class PageControlSwiftExampleViewController: UIViewController, UIScrollViewDeleg
     view.backgroundColor = UIColor.white
 
     let pageColors = [
-      type(of: self).ColorFromRGB(0x55C4f5),
+      type(of: self).ColorFromRGB(0x55C4F5),
```

Comment:
where did we get these colors?

Reply:
They were the colors you guys were already using. Just those two instances the "f" was lower case and I figure you want all the colors to be either all lowercase letters or all uppercase.